src/transformers/visitors/toOrchestrationVisitor.ts

```diff
@@ -304,6 +304,39 @@ const visitor = {
       state.msgSenderParam ??= scope.indicators.msgSenderParam;
       node._newASTPointer.msgSenderParam ??= state.msgSenderParam;
 
+      const initialiseOrchestrationBoilerplateNodes = (fnIndicator: FunctionDefinitionIndicator) => {
+        const newNodes: any = {};
+        const contractName = `${parent.name}Shield`;
+        newNodes.InitialiseKeysNode = buildNode('InitialiseKeys', {
+          contractName,
+          onChainKeyRegistry: fnIndicator.onChainKeyRegistry,
+        });
+        if (fnIndicator.oldCommitmentAccessRequired || fnIndicator.internalFunctionoldCommitmentAccessRequired)
+          newNodes.initialisePreimageNode = buildNode('InitialisePreimage');
+        newNodes.readPreimageNode = buildNode('ReadPreimage', {
+          contractName,
+        });
+        if (fnIndicator.nullifiersRequired || fnIndicator.containsAccessedOnlyState || fnIndicator.internalFunctionInteractsWithSecret ) {
+          newNodes.membershipWitnessNode = buildNode('MembershipWitness', {
+            contractName,
+          });
+          newNodes.calculateNullifierNode = buildNode('CalculateNullifier');
+        }
+        if (fnIndicator.newCommitmentsRequired || fnIndicator.internalFunctionInteractsWithSecret)
+          newNodes.calculateCommitmentNode = buildNode('CalculateCommitment');
+          newNodes.generateProofNode = buildNode('GenerateProof', {
+          circuitName: node.fileName,
+        });
+        newNodes.sendTransactionNode = buildNode('SendTransaction', {
+          functionName: node.fileName,
+          contractName,
+        });
+        newNodes.writePreimageNode = buildNode('WritePreimage', {
+          contractName,
+          onChainKeyRegistry: fnIndicator.onChainKeyRegistry,
+        });
+        return newNodes;
+}
```

Comment:
I think this part was extracted out into `common.ts` more recently for cleanliness - could it go back please? (maybe a merge w master will fix)

Reply:
yes.. did it..